Makefile

```diff
@@ -38,4 +38,4 @@ src/main.o: $(asm_files) $(gfx_files:.png=.2bpp) $(bin_files)
 # Generating a mapfile is required thanks to a bug in rgblink.
 game.gbc: $(obj)
 	rgblink -n $*.sym -m $*.map -o $@ $(obj)
-	rgbfix  -c -n 0 -r 0x03 -s -l 0x33 -k "01" -m 0x1B -j -p 0xFF -t "ZELDA" -v $@
+	rgbfix  -c -n 1 -r 0x03 -s -l 0x33 -k "01" -m 0x1B -j -p 0xFF -t "ZELDA" -v $@
```

Comment:
For the Makefile, if you want to have it build several different revisions, you way want to have a look at how [pokered's Makefile](https://github.com/pret/pokered/blob/master/Makefile) does to build two different version (red and blue).

Basically they have two targets, `red.gbc` and `blue.gbc`, that build the two variants (and also checks the checksum for the two variants).

I guess here we could have `game_rev0.gbc` and `game_rev1.gbc`. `game_rv1.gbc` would build the game with an external symbol `BUILD_REV` defined (using something like `rgbasm -D BUILD_REV r1`), which we could then use to write in the assembly `IF BUILD_REV == "r1"`.